testcontainers/src/core/ports.rs

```diff
@@ -59,11 +71,7 @@ impl TryFrom<PortMap> for Ports {
         let mut ipv6_mapping = HashMap::new();
         for (internal, external) in ports {
             // internal is of the form '8332/tcp', split off the protocol ...
-            let internal_port = if let Some(internal) = internal.split('/').next() {
-                internal.parse()?
-            } else {
-                continue;
-            };
+            let internal_port = internal.parse::<ExposedPort>().expect("Internal port");
```

Comment:
Missed this part, we shouldn't panic anymore.

But don't worry, I'll prepare small refactoring